guide/05-working-with-the-spatially-enabled-dataframe/part3_data_io_writing_data.ipynb

```diff
@@ -0,0 +1,1816 @@
+{
```

Comment:
<ul><li><strong>Caution:</strong>  &nbsp; </li><li>- Memory-based workspaces do not support geodatabase elements, such as feature datasets, representations, topologies, geometric networks, or network datasets.</li><li> &nbsp; - Folders cannot be created in memory-based workspaces.</li><li> &nbsp; - Since memory-based workspaces are stored in your system's physical memory, or RAM, your system may run low on memory if you write large datasets into the workspace. This can negatively impact processing performance.</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1125/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='80'/>